bot/extensions/clashofcode/commands.py

```diff
@@ -0,0 +1,86 @@
+import discord
+from aiohttp import ContentTypeError
+from codingame.http import HTTPError
+from discord import app_commands
+from discord.ext import commands
+
+from bot import core
+from bot.config import settings
+from bot.extensions.clashofcode.utils import coc_client, coc_helper
+from bot.extensions.clashofcode.views import CreateCocView
+
+
+class ClashOfCode(commands.GroupCog, group_name="coc"):
+    def __init__(self, bot: core.DiscordBot):
+        self.bot = bot
+
+        self._create_coc_view = CreateCocView(timeout=None)
+        self.bot.add_view(self._create_coc_view)
+
+    @app_commands.command()
+    @app_commands.default_permissions()
+    async def new(self, interaction: core.InteractionType):
+        """Create a new coc for the session"""
+
+        await interaction.response.send_message(
+            "Select the programming languages and modes you want to use",
+            ephemeral=True,
+            view=self._create_coc_view,
+        )
+
+    @app_commands.command()
+    @app_commands.default_permissions()
+    async def start(self, interaction: core.InteractionType):
```

Comment:
I was thinking differently about this

I was thinking that we start a session and the bot automatically creates a game for us with the provided settings.
Then when that game ends, the bot will automatically create a new game.

Then we can also have commands to force start / stop a clash?